src/components/SidebarMobile/SidebarMobile.jsx

```diff
@@ -93,7 +99,7 @@ export default class SidebarMobile extends React.Component {
     }
 
     return pages.map(page => {
-      let url = `/${page.url}`,
+      let url = `${page.url}`,
```

Comment:
Some of these url changes you made here and in the other sidebar components may be the cause of the travis errors.